stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java

```diff
@@ -194,6 +195,16 @@ private void deletePaths(FileSystem fs, Path basePath) throws Exception {
       return;
     }
 
+    if (mParameters.mClientType == FileSystemClientType.ALLUXIO_HDFS) {
```

Comment:
why do you think this step is necessary? I would think the code below would be more efficient

Reply:
@jja725 I think the previous deletion is for the cloud. On the cloud, we need to delete the subdirectory before the parent directory can be deleted.